include/boost/beast/version.hpp

```diff
@@ -20,7 +20,7 @@
    This is a simple integer that is incremented by one every
    time a set of code changes is merged to the develop branch.
 */
-#define BOOST_BEAST_VERSION 339
+#define BOOST_BEAST_VERSION 340
```

Comment:
This needs to be its own commit, "Set version to 340", and it should be at the top of the log